README.md

```diff
@@ -68,6 +70,31 @@ example_value_count{environment="beta",id="id-A"} 1
 example_value_count{environment="beta",id="id-C"} 3
 ```
 
+Headers & Additional Endpoints
+=============
+An example configuration for multiple endpoints, with headers:
+
+```
+- endpoint: http://example.com/endpoint-1
+  headers:
+  - Content-Type: application/json
```

Comment:
I think better example is:
headers:
- Accept: application/json

As you are not sending any content in Get request. Actually this is something I'm looking for. So, I like your change.